git-credential-oauth.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: git-credential-oauth
+  version: 0.13.4
+  epoch: 0
+  description: "A Git credential helper that securely authenticates to GitHub, GitLab and BitBucket using OAuth"
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - git
+
+environment:
+  environment:
+    CGO_ENABLED: 0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/hickford/git-credential-oauth
+      tag: v${{package.version}}
+      expected-commit: 310d7b668ae9aca58999c95b14f1d95a6db6b045
+
+  - uses: go/build
+    with:
+      ldflags: |
+        -s -w
```

Comment:
```suggestion

```
we can avoid this because we want to preserve `-s` and `-w` is coming from go/build pipeline. ref: https://github.com/chainguard-dev/melange/blob/e788406063b83b86bef24ce16665e82495fc77ff/pkg/build/pipelines/go/build.yaml#L58-L62

Reply:
I have removed this and now only use the `ldlfags` to set the version at link time.